src/reset-password/service.ts

```diff
@@ -0,0 +1,27 @@
+import nodemailer from 'nodemailer';
+import { User } from '../user/types';
+
+export async function sendResetPasswordEmail(user: User) {
+
+    if (!process.env.SMTP_EMAIL_FROM) {
+        throw new Error('The environment variable SMTP_EMAIL_FROM must be set');
+    }
+
+    if (!process.env.SMPT_URL) {
+        throw new Error('The environment variable SMTP_URL must be set. Needs to contain "smtps://[Username]:[Password]@[Host]:[Port]"');
+    }
+
+    const transporter = nodemailer.createTransport(process.env.SMTP_URL);
+
+
+    // send mail with defined transport object
+    const info = await transporter.sendMail({
+        from: '"Admin" <' + process.env.SMTP_EMAIL_FROM + '>', // sender address
```

Comment:
I think it would be even nicer if the entire 'from' string can be taken from SMTP_EMAIL FROM, not just the email address